Index: src/main/java/com/example/demo/Food/recipe/ingredients/Metric.java
===================================================================
diff --git a/src/main/java/com/example/demo/Food/recipe/ingredients/Metric.java b/src/main/java/com/example/demo/Food/recipe/ingredients/Metric.java
deleted file mode 100644
--- a/src/main/java/com/example/demo/Food/recipe/ingredients/Metric.java	(revision bf920200511f8821d5a6f53a6634e4aa5806e837)
+++ /dev/null	(revision bf920200511f8821d5a6f53a6634e4aa5806e837)
@@ -1,26 +0,0 @@
-package com.example.demo.Food.recipe.ingredients;
-
-public class Metric {
-    private String unit;
-    private float value;
-
-    public Metric(){
-
-    }
-
-    public String getUnit() {
-        return unit;
-    }
-
-    public void setUnit(String unit) {
-        this.unit = unit;
-    }
-
-    public float getValue() {
-        return value;
-    }
-
-    public void setValue(float value) {
-        this.value = value;
-    }
-}
Index: src/main/java/com/example/demo/Food/recipe/ingredients/Amount.java
===================================================================
diff --git a/src/main/java/com/example/demo/Food/recipe/ingredients/Amount.java b/src/main/java/com/example/demo/Food/recipe/ingredients/Amount.java
deleted file mode 100644
--- a/src/main/java/com/example/demo/Food/recipe/ingredients/Amount.java	(revision bf920200511f8821d5a6f53a6634e4aa5806e837)
+++ /dev/null	(revision bf920200511f8821d5a6f53a6634e4aa5806e837)
@@ -1,16 +0,0 @@
-package com.example.demo.Food.recipe.ingredients;
-
-public class Amount {
-    private Metric metric;
-    public Amount(){
-
-    }
-
-    public Metric getMetric() {
-        return metric;
-    }
-
-    public void setMetric(Metric metric) {
-        this.metric = metric;
-    }
-}
Index: src/main/java/com/example/demo/Food/recipe/ingredients/IngredientDescription.java
===================================================================
diff --git a/src/main/java/com/example/demo/Food/recipe/ingredients/IngredientDescription.java b/src/main/java/com/example/demo/Food/recipe/ingredients/IngredientDescription.java
deleted file mode 100644
--- a/src/main/java/com/example/demo/Food/recipe/ingredients/IngredientDescription.java	(revision bf920200511f8821d5a6f53a6634e4aa5806e837)
+++ /dev/null	(revision bf920200511f8821d5a6f53a6634e4aa5806e837)
@@ -1,47 +0,0 @@
-package com.example.demo.Food.recipe.ingredients;
-
-import com.example.demo.Food.FoodController;
-import com.example.demo.Food.FoodService;
-import com.example.demo.Food.ingredient_id.IngredientId;
-
-public class IngredientDescription {
-    private int id;
-
-    private static int foodId = 161;
-
-    public int getId() {
-        return id;
-    }
-
-    public void setId(int id) {
-        if (id == 0){
-            this.id = foodId ;
-            foodId ++;
-        }
-        else{
-            this.id = id;
-        }
-
-    }
-
-    private String name;
-    private Amount amount;
-    public IngredientDescription(){
-    }
-
-    public String getName() {
-        return name;
-    }
-
-    public void setName(String name) {
-        this.name = name;
-    }
-
-    public Amount getAmount() {
-        return amount;
-    }
-
-    public void setAmount(Amount amount) {
-        this.amount = amount;
-    }
-}
Index: src/main/java/com/example/demo/Food/recipe/ingredients/Ingredients.java
===================================================================
diff --git a/src/main/java/com/example/demo/Food/recipe/ingredients/Ingredients.java b/src/main/java/com/example/demo/Food/recipe/ingredients/Ingredients.java
deleted file mode 100644
--- a/src/main/java/com/example/demo/Food/recipe/ingredients/Ingredients.java	(revision bf920200511f8821d5a6f53a6634e4aa5806e837)
+++ /dev/null	(revision bf920200511f8821d5a6f53a6634e4aa5806e837)
@@ -1,19 +0,0 @@
-package com.example.demo.Food.recipe.ingredients;
-
-import java.util.ArrayList;
-
-public class Ingredients {
-    private ArrayList<IngredientDescription> ingredients;
-
-    public Ingredients(){
-
-    }
-
-    public ArrayList<IngredientDescription> getIngredients() {
-        return ingredients;
-    }
-
-    public void setIngredients(ArrayList<IngredientDescription> ingredients) {
-        this.ingredients = ingredients;
-    }
-}
